user-guide/Getting_started/Installing_a_DMA/Using_a_pre_installed_DataMiner_VHDX.md

```diff
@@ -0,0 +1,155 @@
+---
+uid: Using_a_pre_installed_DataMiner_VHDX
+---
+
+# Using a pre-installed DataMiner Virtual Hard Disk
+
+You can download a Virtual Hard Disk (VHDX) with DataMiner pre-installed to immediately get started.
```

Comment:
An important piece of information that is missing here is where the VHDX can be downloaded. Could you add this?

Reply:
A link was added to the page were you can find the downloads